src/access/controllers/data-access-amendment-edit-controller.js

```diff
@@ -34,7 +34,7 @@
       DataAccessEntityResource.get($scope.entityUrl, $routeParams.id) :
       {
         'obiba.mica.DataAccessAmendmentDto.amendment': { parentId: $routeParams.parentId },
-        $promise: new Promise(function (resolve) { setTimeout(resolve, 0, {}); }),
+        $promise: DataAccessEntityResource.get(DataAccessEntityUrls.getDataAccessRequestBaseUrl(), $routeParams.parentId).$promise,
```

Comment:
The DAR form may be very different and you do not want a amendment model with values that do not match any fields in the amendment schema form.

Reply:
Then I don't think we should add any fields at all.
From the CAG's amendment form it seems that the applicant must first define the "Nature of modifications" before attempting any such modifications, thus, imo, amendment must be left blank by default.